README.md

```diff
@@ -73,8 +73,22 @@ Under your team directory:
 - In *__data.tf__*, edit the data block changing the *__key_id__* to match the new key to be used in the pipeline.\
 *__default key:__* `key_id = "arn:aws:kms:eu-west-2:${data.aws_caller_identity.current.account_id}:alias/ebs-encryption-key"`\
 *__team key:__* `key_id = "arn:aws:kms:eu-west-2:${local.environment_management.account_ids["<team environemnt>"]}:alias/<team kms key alias name>"`\
+
 *__example of team key:__* `key_id = "arn:aws:kms:eu-west-2:${local.environment_management.account_ids["sprinkler-development"]}:alias/sprinkler_ebs-encryption-key"`
 
+More details of the required change can be seen in https://github.com/ministryofjustice/modernisation-platform-ami-builds/pull/18/files/6a589a6d3d0dc70f2bc28cb8cbb84075cad9d73c - ignore the rhel7 sections. \
```

Comment:
I think there is too much information in this PR for it to be useful as an example here